getting-started/templates/systemlink-values.yaml

```diff
@@ -1008,6 +1008,35 @@ argoworkflows:
 ##
 nbexecservice:
   maxNumberOfWorkflowsToSchedule: *workflowParallelism
+  ## Configure S3/MinIO access.
+  ##
+  s3:
+    ## Secret name for S3 credentials.
+    ##
+    secretName: "executions-s3-credentials"
+    ## S3 access type. Supported values are "ACCESS_KEY" and "AWS_WEB_IDENTITY_TOKEN".
+    ##
+    authType: "ACCESS_KEY"
```

Comment:
Should not be necessary, should be preconfigured with this. I'm inclined to keep consistency with feedservice

Reply:
Removed